packages/core/src/cli/validate.ts

```diff
@@ -151,6 +155,10 @@ function getUnsafeAllowKinds(unsafeAllow: string | undefined): ValidationError['
   return unsafeAllowTokens as errorKindsType[];
 }
 
+function getReferenceBuildInfoDirsArray(referenceBuildInfoDirs: string | undefined): string[] | undefined {
+  return referenceBuildInfoDirs !== undefined ? referenceBuildInfoDirs.split(/,+/) : undefined;
```

Comment:
Does that work with typescript type system ?

```suggestion
  return referenceBuildInfoDirs?.split(/,+/);
```